### PR TITLE
Patch libnetcdf pin

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @isuruf

### CODE REVIEW

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -498,6 +498,9 @@ def _gen_new_index(repodata, subdir):
         if any(dep.startswith("gf2x") for dep in deps):
             _pin_stricter(fn, record, "gf2x", "x.x")
 
+        if any(dep.startswith("libnetcdf >=4.7.3") for dep in deps):
```
Comment (isuruf) [inline recipe/gen_patch_json.py]: @ocefpaf, is it okay to fix only packages built with `4.7.3`?

Comment (ocefpaf) [reply recipe/gen_patch_json.py]: Maybe >=4.7.0 to be safe? B/c old build with 4.7.{0,1,2} will install 4.7.4 too, right?

Comment (ocefpaf) [reply recipe/gen_patch_json.py]: What do you think? Does that makes sense?

Comment (isuruf) [reply recipe/gen_patch_json.py]: There are no builds with 4.7.0, 4.7.2. `4.7.1` is pinned to `x.x.x`

Comment (ocefpaf) [reply recipe/gen_patch_json.py]: Great. So let's get this one merge then!

**Comment by ocefpaf** [comment]
Thanks @isuruf!

**Pull request merged**